pontoon/machinery/utils.py

```diff
@@ -61,6 +79,45 @@ def get_google_translate_data(text, locale_code):
         }
 
 
+def get_google_automl_translation(text, locale):
+    try:
+        client = translate.TranslationServiceClient()
+    except DefaultCredentialsError as e:
+        log.error("Google AutoML Translation error: {e}")
+        return {
+            "status": False,
+            "message": f"{e}",
+        }
+
+    project_id = "85591518533"
+    model_id = locale.google_automl_model
+    location = "us-central1"
+    parent = f"projects/{project_id}/locations/{location}"
+    model_path = f"{parent}/models/{model_id}"
+
+    response = client.translate_text(
+        request={
+            "contents": [text],
+            "target_language_code": locale.google_translate_code,
+            "model": model_path,
+            "source_language_code": "en",
```

Comment:
Just to check, is this always the case in Pontoon?

Reply:
Yeah, and the same value is [hardcoded](https://github.com/mozilla/pontoon/blob/24a18c0268a1d9d80968734f7465c8002af5dd19/pontoon/machinery/utils.py#L51) in the generic API.

We track support for other source locales in https://github.com/mozilla/pontoon/issues/2052.